lib/export_dsi_approvers_to_big_query.rb

```diff
@@ -16,28 +16,30 @@ def run!
   def present_for_big_query(batch)
     batch.map do |user|
       {
-        user_id: user["userId"],
-        role_id: user["roleId"],
-        role_name: user["roleName"],
         email: user["email"],
         family_name: user["familyName"],
         given_name: user["givenName"],
+        la_code: la_code(user),
+        trust_uid: user.dig("organisation", "uid"),
+        role_id: user["roleId"],
+        role_name: user["roleName"],
         school_urn: user.dig("organisation", "urn"),
-        organisation_uid: user.dig("organisation", "uid"),
+        user_id: user["userId"],
```

Comment:
Changes here:
- sort alphabetically
- rename organisation_uid to trust_uid
- add la_code